keyboards/sol/common/ssd1306.h

```diff
@@ -0,0 +1,93 @@
+#ifndef SSD1306_H
+#define SSD1306_H
+
+#include <stdbool.h>
+#include <stdio.h>
+#include "pincontrol.h"
+#include "config.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```